github/ci/prow-deploy/files/jobs/kubevirt/kubevirt.github.io/kubevirt-github-io-presubmits.yaml

```diff
@@ -13,3 +13,23 @@ presubmits:
                 value: "true"
             command: ["/bin/sh", "-c"]
             args: ["/usr/local/bin/bundle install && bundle exec rake"]
+    - name: kubevirt-io-presubmit-markdown-linter
+      decorate: true
+      always_run: true
+      skip_report: false
+      cluster: ibm-prow-jobs
+      spec:
+        containers:
+          - image: quay.io/kubevirtci/ruby:v20210628-647402e
+            command: ["/bin/sh", "-c"]
+            args: ["/bin/make check_lint"]
+    - name: kubevirt-io-presubmit-spell-checker
```

Comment:
Ooof!  I didn't notice this...  Could you separate this to a different pull req?